server/encoder.js

```diff
@@ -0,0 +1,27 @@
+const robot = require('robotjs');
+
+const posInputs = {
+	'ButtonA': 'l',
+    'ButtonB': 'k',
+    'ButtonStart': 'enter',
+    'ButtonSelect': 'space',
+    'ButtonUp': 'w',
+    'ButtonDown': 's',
+    'ButtonLeft': 'a',
+    'ButtonRight': 'd'
+}
+
+function encoder(data) {
+
+	if (data.player == 1) {
```

Comment:
why did you choose this instead what there was?
this looks like it is a more complex solution and less enginerd solution than the object based one.
are you familiar with how the system works?